# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1320396940 14400
#      Fri Nov 04 04:55:40 2011 -0400
# Node ID d3f0d75faf2c46b382aca5a0705e4a008b9ffd25
# Parent  240d6810204cbcc0ef1e949753f7de29cfb82684
title: avoid spurious output

* title.m: Rename output argument to retval from h.

diff --git a/scripts/plot/title.m b/scripts/plot/title.m
--- a/scripts/plot/title.m
+++ b/scripts/plot/title.m
@@ -21,17 +21,17 @@
 ## @deftypefnx {Function File} {} title (@var{string}, @var{p1}, @var{v1}, @dots{})
 ## @deftypefnx {Function File} {} title (@var{h}, @dots{})
 ## @deftypefnx {Function File} {@var{h} =} title (@dots{})
 ## Create a title object and return a handle to it.
 ## @end deftypefn
 
 ## Author: jwe
 
-function h = title (varargin)
+function retval = title (varargin)
 
   [h, varargin, nargin] = __plt_get_axis_arg__ ("title", varargin{:});
 
   if (rem (nargin, 2) != 1)
     print_usage ();
   endif
 
   tmp = __axis_label__ (h, "title", varargin{:});
